tests/common/platform/interface_utils.py

```diff
@@ -44,6 +44,9 @@ def parse_intf_status(lines):
 
 
 def check_interface_status_of_up_ports(duthost):
+    if duthost.is_supervisor_node():
```

Comment:
in cisco chassis sup, fabric links are considered interfaces.  let's also check more on current usages of this utility.

Reply:
Given that the major problem is on the virtual chassis supervisor, we can check whether the platform is VS. If so, we skip the supervisor, otherwise we proceed normally.

Reply:
shoudl the checking from the caller side , ie virtual chassis side, and not this utility?  the utility is to check and return interface status.  Returning true for virtual chassis sup is also misleading.